### PR TITLE
chore: update cypress v10

### PR DESCRIPTION
**Pull request by celine-s**

### CODE REVIEW

File: cypress.config.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    retries: {
+        runMode: 2,
+    },
+    component: {
+        setupNodeEvents() {
+            return require('./cypress/plugins/index.ts');
+        },
+        viewportHeight: 1280,
+        viewportWidth: 800,
+        specPattern: ['./src//**/*.cy.tsx', './**/*.spec.ts', './src//**/*.spec.tsx'],
```
Comment (julianiff) [inline cypress.config.ts:14]: /src// the 2 // are intented?

File: tsconfig.json [hunk 2]
```diff
@@ -24,6 +25,6 @@
         },
         "types": ["cypress", "cypress-real-events", "vite/client"]
     },
-    "include": ["src/types", "src/**/*"],
+    "include": ["src/types", "src/**/*", "cypress/support/component.ts"],
```
Comment (julianiff) [inline tsconfig.json:28]: why does component.ts need to be included in the output build?

**Review by julianiff — APPROVED** [review]

File: cypress.config.ts (hunk 1 above)
Comment (celine-s) [reply cypress.config.ts]: oh, no - thanks. Adapted in [ced93cc](https://github.com/Frontify/fondue/pull/881/commits/ced93cc13033c3e3660257880031d56da99aa50d)

File: tsconfig.json (hunk 2 above)
Comment (celine-s) [reply tsconfig.json:28]: in the component the typing for the `mount` command is defined. That the test file know of this `cy.mount` it has to be included there. Would it be better/clearer if I would make a file `cypress/support/types.ts`?

Comment (julianiff) [reply tsconfig.json:28]: Nono i was just curious to understand the include :) Thx for clarifying :)

**Pull request merged**